src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -35,6 +35,13 @@ object WdlExpression {
       ast.isFunctionCall
         && ast.params.size == 1
         && WdlFunctionsWithSingleFileParameter.contains(ast.functionName))
+
+    def isGlobFunctionCall: Boolean = (
+      ast.isFunctionCall
```

Comment:
This is well below the limits of pain for our line length limit czar, it can be made into a one liner